crates/typst-cli/src/world.rs

```diff
@@ -70,12 +74,21 @@ impl SystemWorld {
         let main_path = VirtualPath::within_root(&input, &root)
             .ok_or("input file must be contained in project root")?;
 
+        // Convert the Vec<(String, String)> to a Typst Dictionary
+        let inputs = command
+            .plain_inputs
+            .iter()
+            .map(|(k, v)| (k.clone().into(), Value::Str(v.clone().into())));
+        let inputs = Dict::from_iter(inputs);
+
+        let sys_args = SysArguments { inputs };
+
         Ok(Self {
             workdir: std::env::current_dir().ok(),
             input,
```

Comment:
```suggestion
            source,
```

Reply:
Should I also rename the function called `input` which returns a reference to `SystemWorld`'s input/source to `source` (line 133)?
If so, it would clash with the existing function `source` to implement `World` for `SystemWorld` (line 162).